### ISSUE TITLE
I don't understand how to create a file and to access it

### ISSUE THREAD

**Issue by pedefe** [android-js/androidjs#235]
Hello,

I see the API app.getPath() which allow to know some specific paths of smartphone.
I call app.getPath('userData') in my web page (front app), and I get "/storage/emulated/0/Android/data/com.androidjs.webview/files"

But for me the web-page cannot write directly in a file.... For exemple, 'fs' is not available....

On over side, how the 'back' side (the nodejs process) can use app.getPath() in order to know the paths of the smartphone ?

Thank for help.

**Comment by Chhekur** [comment]
@pedefe correct, so what you can do is invoke `app.getPath()` in front and send it back via `IPC` and write the data to the file.
